dataland-frontend/src/components/resources/landing/LandingLogin.vue

```diff
@@ -49,21 +49,22 @@
         <div class="col-6 col-offset-3">
           <Card class="d-card">
             <template #title>
-              <h2 class="text-gray-100 text-left">Access EU Taxonomy Data</h2>
+              <h2 class="text-gray-100 text-left">Join Dataland to access our data</h2>
             </template>
             <template #content>
               <p class="text-gray-800 text-left">
-                Preview Dataland by looking at <strong>EU Taxonomy</strong> data from more than <strong>300</strong> Germany public companies.</p>
+                Register free to access Eu Taxonomy data from more than <strong>300</strong> Germany public companies.</p>
               <div class="grid">
-                <div class="col-3 col-offset-4 p-fluid pl-0">
-                  <router-link to="/searchtaxonomy" class="no-underline">
-                  <Button class="uppercase p-button p-button pl-2 pr-1 pb-1 pt-1 justify-content-center h-2rem w-full" name="get_started_button">
+                <div class="col-9 p-fluid pr-0">
+                  <InputText type="text" placeholder="Email address" class="h-2rem" name="email_input_landing" v-on:input="setEmail"/>
+                </div>
+                <div class="col-3 p-fluid pl-0">
+                  <Button class="uppercase p-button p-button pl-2 pr-1 pb-1 pt-1 justify-content-center h-2rem w-full" name="get_started_button" @click="login(email)" >
```

Comment:
Ich fände 
```
@click="login"
```
besser, die funktion holt sich dann selbt `this.email`.

Reply:
restyled to register button in accordance with new figma